server/routes/bookingsRoute.js

```diff
@@ -47,4 +49,52 @@ router.post('/students/:student_id/bookings', lectureValidation.checkLecture(),v
         res.status(400).json(error);
     }
 })
+
+
+/**
+ * @swagger
+ * //students/{student_id}/delete_booking:
+ *  post:
+ *    tags:
+ *      - students
+ *    summary: "Delete a new booking"
+ *    description: "A student can delete a lecture "
+ *    parameters:
+ *      - in: path
+ *        name: student_id
+ *        required: true
+ *        schema:
+ *          type: integer
+ *          format: int64
+ *    responses:
+ *       "201 and num=1":
+ *         description: "Successful deletion"
+ *         schema:
+ *           type: "object"
+ *       "201 and num=0":
+ *         description: "Nothing to delete"
+ *         schema:
+ *           type: "object"
+ *       "400":
+ *         description: "Invalid status value"
+ *    security:
+ *     - petstore_auth:
+ *       - "write:pets"
+ *       - "read:pets"
+ */
+router.post('/students/:student_id/delete_booking', lectureValidation.checkLecture(),validator, async(req,res) =>{
+    const {lecture_id} = req.body;
+    const student_id= +req.params.student_id;
+    const datetime= moment();
```

Comment:
Should the moment object be converted into an agreed `.format` string?

Reply:
Yes, for now I've formatted with .format('YYYY-MM-DD HH:mm'). Let me know if we like something more specifical